src/analyzer/protocol/asn1/asn1.pac

```diff
@@ -87,10 +87,18 @@ type Array = record {
 
 ############################## ASN.1 Conversion Functions
 
+# Converts an 8-byte string into an int64. If this string is longer than
+# 8 bytes, it throws an exception.
```

Comment:
This should be "a weird", right? :)

Reply:
Fixed. I'm going to let CI chew on this and merge it if everything passes.